tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py

```diff
@@ -1084,6 +1084,76 @@ def testGLSTMCell(self):
                             h_res, 1e-5)
 
 
+class LayerNormBasicGRUCellTest(test.TestCase):
+
+  # NOTE: all the values in the current test case have been calculated.
+
+  def testMultiCellLayerNormBasicGRUCell(self):
+    with self.test_session() as sess:
+      with variable_scope.variable_scope("root", initializer=init_ops.Identity()):
+        x = array_ops.zeros([1, 3])
+        h0 = array_ops.zeros([1, 3])
+        h1 = array_ops.zeros([1, 3])
+        state = (h0, h1)
+        single_cell = lambda: contrib_rnn_cell.LayerNormBasicGRUCell(3)
+        cell = rnn_cell.MultiRNNCell([single_cell() for _ in range(2)], state_is_tuple=True)
+
+        g, out_m = cell(x, state)
+        sess.run([variables.global_variables_initializer()])
+        res = sess.run([g, out_m], {
+          x.name: np.array([[0, 1, 2]]),
+          h0.name: 0.1 * np.asarray([[3, 4, 5]]),
+          h1.name: 0.1 * np.asarray([[6, 7, 8]])
+        })
+
+        expected_state0 = np.array([[0.0261896, 0.2, 0.8666091]])
+        expected_state1 = np.array([[0.18898079, 0.26299422, 0.94606379]])
+
+        actual_h = res[0]
+        actual_state0 = res[1][0]
+        actual_state1 = res[1][1]
+
+        self.assertAllClose(expected_state1, actual_h, 1e-5)
```

Comment:
use the kwarg for 1e-5.  it's not clear what this argument is.  is it atol?  rtol?

Reply:
done